catkin_home/src/action_selectors/scripts/parser.py

```diff
@@ -64,9 +71,11 @@ def callback(msg):
     #open list of actions
     #call parser
     #send message to engine 
+    pub_resp = rospy.Publisher('BotResponse', response, queue_size=10)
```

Comment:
Is creating this again and again in each callback okay?

Reply:
It is not, i will fix it.